test/utils.ts

```diff
@@ -71,27 +71,27 @@ test('getDiscussionNumber', t => {
 	]);
 	for (const [url, result] of pairs) {
 		location.href = url;
-		t.is(result, utils.getDiscussionNumber(url));
+		t.is(result, utils.getDiscussionNumber());
 	}
 });
 
 test('getRepoPath', t => {
-	const pairs = new Map([
+	const pairs = new Map<string, boolean|string>([
 		[
 			'https://github.com',
-			false
+			undefined
```

Comment:
Comparing to `undefined` now because this got changed in `getRepoPath` to reflect a return type of `string?` instead of `string|boolean`.